src/main/java/pages/hero.java

```diff
@@ -10,9 +10,9 @@
 import org.openqa.selenium.support.ui.WebDriverWait;
 import utils.CommonFuntions;
 
-public class HomePageFS {
+public class hero {
```

Comment:
Class names should start with capital letters as per Java Naming Conventions: Lets read this one (both of us) and we can have a short discussion on our next session: https://www.geeksforgeeks.org/java-naming-conventions

Reply:
Done, thanks for sharing the link https://www.cdn.geeksforgeeks.org/java-naming-conventions/